src/drawing/backend_impl/bitmap.rs

```diff
@@ -228,6 +228,7 @@ impl<'a> BitMapBackend<'a> {
             .collect()
     }
 
+    #[allow(clippy::cast_ptr_alignment, clippy::many_single_char_names)]
```

Comment:
Just realized recently there's a huge bitmap backend refactoring, that causing some conflict. As long as this file is only fixing clippy issues, so would you mind remove this file from the PR, thus we have less conflict and I want to run clippy again when I get ready to merge :) - Thanks!